zanata-war/src/test/java/org/zanata/webtrans/test/GwtTestSample.java

```diff
@@ -16,6 +16,6 @@ public void testSomething() {
         // avoided wherever possible in favour of plain JUnit tests.
         // Only code that requires a DOM should be tested with these
         // (e.g. most view classes).
-        Assert.assertTrue(true);
+        Assertions.assertThat(true).isTrue();
```

Comment:
Maintaining useless code is always fun.  If we want to keep this line at all, we should just switch from `junit.framework.Assert` to `org.junit.Assert`, instead of making this code depend on assertj.  But really, just delete the line.  In fact, delete the whole useless test class.